### PR TITLE
Smd-231-update-upload-screen

### PR DESCRIPTION
**Pull request by bobbi-hbrown**
https://dluhcdigital.atlassian.net/jira/software/c/projects/SMD/boards/140?quickFilter=815&selectedIssue=SMD-231

### Change description
Updates the copy displayed in different scenarios for the due date.

Due in more than several days:

Screenshot 2023-10-11 095857

Due tomorrow:

Screenshot 2023-10-11 100028

Due today:

Screenshot 2023-10-11 100121

Overdue:

Screenshot 2023-10-11 100233

### How to test
The environment variable SUBMIT_DEADLINE can be changed on line 33 of the file config/envs/default.py (when testing locally).

### CODE REVIEW

File: app/templates/main/upload.html [hunk 1]
```diff
@@ -40,33 +56,80 @@ <h1 class="govuk-heading-l">Get ready to submit your monitoring and evaluation d
...
-                    <h2 class="govuk-heading-m">Upload your return for the April to September period</h2>
-                    <p class="govuk-body">You are uploading for: <b>{% for la in local_authorities %}<li class="govuk-body">{{ la }}</li>{% endfor %}</b> </p>
-                    <p class="govuk-body">You need to download the (LINK)[spreadsheet template]   so that you can complete the current return.</p>
-                    <br>
-                    <br>
+                    {% if local_authorities|length > 1 %}
+                        {% set localAuthorities %} {{ ", ".join(local_authorities) }} {% endset %}
+                    {% else %}
+                        {% set localAuthorities %} {{ " ".join(local_authorities) }} {% endset %}
+                    {% endif %}
```
Comment (gidsg) [inline app/templates/main/upload.html:63]: Bit confused about what this if statement is doing? Seems like the same code runs in both cases? Sorry if I missed something?

**Review by cyrusdobbs — CHANGES_REQUESTED** [review]

File: app/templates/main/upload.html (hunk 1 above)
Comment (nsnape) [reply app/templates/main/upload.html:63]: there's a slight difference in the join method Gids

File: app/templates/main/upload.html [hunk 2]
```diff
@@ -40,33 +56,80 @@ <h1 class="govuk-heading-l">Get ready to submit your monitoring and evaluation d
...
 
-                    <h2 class="govuk-heading-m">Upload your return for the April to September period</h2>
-                    <p class="govuk-body">You are uploading for: <b>{% for la in local_authorities %}<li class="govuk-body">{{ la }}</li>{% endfor %}</b> </p>
-                    <p class="govuk-body">You need to download the (LINK)[spreadsheet template]   so that you can complete the current return.</p>
-                    <br>
-                    <br>
+                    {% if local_authorities|length > 1 %}
+                        {% set localAuthorities %} {{ ", ".join(local_authorities) }} {% endset %}
+                    {% else %}
+                        {% set localAuthorities %} {{ " ".join(local_authorities) }} {% endset %}
```
[the remark this answers is not in the document]
Comment (bobbi-hbrown) [reply app/templates/main/upload.html:62]: I thought contractors would have access to and therefore see more than one, like they do in the current example? I can remove this if that's no longer the case

**Review by Joel-Pearce — APPROVED** [review]
LGTM

File: app/templates/main/upload.html (hunk 1 above)
Comment (gidsg) [reply app/templates/main/upload.html]: If there's only 1 I don't think we need a join at all, can we refactor this?

Comment (gidsg) [reply app/templates/main/upload.html]: Seems like as it is, it will add an erroneous whitespace?

File: app/templates/main/upload.html [hunk 3]
```diff
@@ -40,33 +56,76 @@ <h1 class="govuk-heading-l">Get ready to submit your monitoring and evaluation d
...
+                            {
+                              "text": "Fund"
+                            },
+                            {
+                              "html": fund
+                            }
+                          ],
+                          [
+                            {
+                              "text": "Local authority"
```
Comment (gidsg) [inline app/templates/main/upload.html:84]: I think this needs to be pluralised in the case there are multiple authorities.

File: app/templates/main/upload.html (hunk 1 above)
Comment (cyrusdobbs) [reply app/templates/main/upload.html]: I mentioned this and it's been refactored 👍

**Review by cyrusdobbs — APPROVED** [review]
Nice work, LGTM 👍

**Pull request merged**